prow/cmd/tide/main.go

```diff
@@ -70,6 +72,8 @@ type options struct {
 
 	// Gerrit-related options
 	cookiefilePath string
+	githubOnly     bool
```

Comment:
Nit: Instead of a bool, maybe use a string? And only accept either "github" (default) or "gerrit".

Or maybe the flag handling lib is smart enough to understand Go's iota stuff for enum consts? IDK

Reply:
it doesn't really make that much difference imo. As mentioned at the flags section, these two flags only make sense when both GitHub and Gerrit are supported.

tbh these two flags are technically not needed. Majority of the times we could make decisions based on tide configurations.